packages/cashier/src/Components/cashier-container.jsx

```diff
@@ -5,8 +5,15 @@ import { routes } from '@deriv/shared';
 import { Button, Loading, Text } from '@deriv/components';
 import { Localize, localize } from '@deriv/translations';
 
-const CashierContainer = ({ iframe_height, iframe_url, is_crypto, is_loading }) => {
+const CashierContainer = ({ iframe_height, iframe_url, iframeWillMount, is_crypto, is_loading }) => {
     const history = useHistory();
+
+    React.useEffect(() => {
+        return () => {
+            iframeWillMount();
+        };
+    }, [iframeWillMount]);
```

Comment:
I'm a little confused about this. What do you want to achieve with this?

Reply:
@carol-binary Since both fiat deposit and withdraw use this component, I need to make the iframe empty in `componentWillMount`

Reply:
The return is executed when the component is unmounted. Currently, you're tracking `iframeWillMount` only to execute it when the component is unmounted. So you're making the iframe empty in `componentWillUnmount`.

Reply:
@carol-binary Yeah, I meant that

Reply:
Umm in that case `iframeWillMount` doesn't sound right actually, what do you think?

Reply:
@carol-binary Let me change the name to `clearIframe`